src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -167,6 +167,7 @@ public ForgeMod()
 
         MinecraftForge.EVENT_BUS.addListener(VillagerTradingManager::loadTrades);
         MinecraftForge.EVENT_BUS.register(MinecraftForge.INTERNAL_HANDLER);
+        MinecraftForge.EVENT_BUS.addListener(this::mappingChanged);
```

Comment:
Should the license header of this file also be updated with the new year, given that you have modified it?